pages/linux/ncdu.md

```diff
@@ -0,0 +1,19 @@
+# ncdu
+
+> Disk usage analyzer with an ncurses interface.
+
+- Analyze current working directory:
+
+`ncdu`
+
+- Analyze given directory:
+
+`ncdu directory/`
+
+- Save results to file:
+
+`ncdu -o file`
+
+- Exclude files that match pattern (for ex. *.txt), argument can be given multiple times to add more patterns:
```

Comment:
Perhaps ....`that match a pattern`.... would read better?

Also, I think a concrete example might work better here. What about changing `PATTERN` in the example below to `'{{*.txt}}'` and removing the bit in brackets? I assume that the pattern would need to be in single quotes in order to avoid glob expansion by the shell.

Reply:
I am not sure about the brackets, I just tested and it takes it with or without, but mazbe in other use cases thez might be needed so I am adding them. Otherwise done as requested.